src/Table/Footer.jsx

```diff
@@ -0,0 +1,149 @@
+import classnames from "classnames";
+import React, {Component, PropTypes} from "react";
+
+import {Button} from "../Button/Button";
+import Cell from "./Cell";
+
+require("./Footer.less");
+
+
+/**
+ * Pagination footer for the Table component.
+ * Only rendered if there are more than 1 page of data available.
+ */
+export default class Footer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+    };
+  }
+
+  _selectPage(page) {
+    if (page === this.props.currentPage) {
+      return;
+    }
+
+    this.props.onPageChange(page);
+  }
+
+  _renderEllipsis() {
+    return <span className={Footer.cssClass.ELLIPSIS}>{"•••"}</span>;
```

Comment:
yeah, just wanted something a little more prominent than the default. switched back to hellip and bumped up the font-size instead.